src/App/tokenStats/CryptoStats/stats-getter.service.ts

```diff
@@ -13,14 +13,11 @@ class StatsGetterService {
     private configService: ConfigService,
   ) {}
 
-  private cmcApiCall(name: string) {
-    // TEMP HACK
-    const cmcName = (name === 'binancecoin') ? 'bnb' : name;
-
+  private cmcApiCall(name?: string) {
```

Comment:
this can't be nullable

```suggestion
  private cmcApiCall(name: string) {
```